BentoKit/BentoKit/Components/TitledDescription.swift

```diff
@@ -508,26 +508,50 @@ private extension Component.TitledDescription.View {
 }
 
 public extension Component.TitledDescription {
+    public final class ContentStyleSheet: ViewStyleSheet<BaseStackView> {
+        public var spacing: CGFloat
+        public var distribution: UIStackView.Distribution
+        public var alignment: UIStackView.Alignment
+        public var isLayoutMarginsRelativeArrangement: Bool
 
-    public final class StyleSheet: BaseViewStyleSheet<View> {
+        public init(
+            spacing: CGFloat = 16,
+            distribution: UIStackView.Distribution = .fill,
```

Comment:
I don't think distributions should be customisable. Plenty of them violates the algorithm in the height computer.

Reply:
It was customisable before that's why I left it

Reply:
Ah, no you're right it was not used before